### ISSUE TITLE
Lint/UnneededCopDisableDirective seems to change depending on absolute/relative path for "-c" and "-s" flags

### ISSUE THREAD

**Issue by ndbroadbent** [rubocop-hq/rubocop#6588]
I also opened this issue in rubocop-rails: https://github.com/rubocop-hq/rubocop-rails/issues/23

I created a repo here that can reproduce the issue: https://github.com/ndbroadbent/rubocop-bug

## Expected behavior

`Lint/UnneededCopDisableDirective` should not have any false positives.

## Actual behavior

`Lint/UnneededCopDisableDirective` has a false positive that seems to depend on the path provided to both the `-c` and `-s` flags.

## Steps to reproduce the problem

Clone this repo into `/tmp/rubocop-bug`: https://github.com/ndbroadbent/rubocop-bug

Then run `./script.sh`

## RuboCop version

```
0.61.1 (using Parser 2.5.3.0, running on ruby 2.5.3 x86_64-darwin18)
```

**Comment by ndbroadbent** [comment]
I think I just noticed the same issue for `Style/StringLiterals`. It shows a linting error in my text editor (VS Code), but no error when I run `rubocop` on the command line.

The `vscode-ruby` extension copies ruby files to a temporary file, and runs Rubocop with the `-c` flag

**Comment by ndbroadbent** [comment]
@jaredbeck, thanks, I think you might be right!

This caused the error to disappear:

```yaml
Rails/Output:
  Include:
    - "**/*.rb"
```

And when I comment it out:

```
$ rubocop -c ./.rubocop.yml -s /tmp/test_model.rb
Inspecting 1 file
W

Offenses:

/tmp/test_model.rb:6:5: W: Lint/UnneededCopDisableDirective: Unnecessary disabling of Rails/Output.
    # rubocop:disable Rails/Output
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

1 file inspected, 1 offense detected
```


So yes, maybe this is a problem with the vs-code-ruby extension. It's copying the original Ruby file into a temp file, and then passing that filename to the `-s` flag, but this breaks the `Rails/Output` rule because the path doesn't match. Does the `-a` autocorrect option also work with `-s`? That might be part of the issue as well.
